problems/countThrees.js

```diff
@@ -1,20 +1,26 @@
 /**
-* Counts the number of threes in a given number.
-* @param {number} n - The input number.
-* @returns {number} - Returns how many threes are in the input.
-*
-* ex: countThrees(300)
-* returns: 1
-*
-* ex: countThrees(3203)
-* returns: 2
-*
-* ex: countThrees(47)
-* returns: 0
-*/
-
-function countThrees() {
+ * Counts the number of threes in a given number.
+ * @param {number} n - The input number.
+ * @returns {number} - Returns how many threes are in the input.
+ *
+ * ex: countThrees(300)
+ * returns: 1
+ *
+ * ex: countThrees(3203)
+ * returns: 2
+ *
+ * ex: countThrees(47)
+ * returns: 0
+ */
 
+function countThrees(n) {
+    let counts = {}
+  for (let i = 0; i < arr.length; i++) {
```

Comment:
We don't have an arr. It doesn't exist. Instead we have a number. We can convert the number into a string with toString and then iterate through that string.